crates/dht/src/hash.rs

```diff
@@ -0,0 +1,443 @@
+//! Partition of the hash space.
+//!
+//! The space of possible hashes is mapped to a 32-bit location. See [::kitsune2_api::id::Id::loc] for
+//! more information about this. Each agent is responsible for storing and serving some part of
+//! the hash space. This module provides a structure that partitions the hash space by some factor.
+//!
+//! The factor must be chosen up front and cannot be changed. The factor determines the number of
+//! partitions to create. A factor of 32 will create a single partition. Each unit decrease from 32
+//! will double the number of partitions. A good default for a network that is expected to grow to
+//! a few thousand nodes is 21. This will create 2^11 partitions, which is 2048. Given that data is
+//! also stored redundantly, say 10 times, this means that 20,000 nodes are needed for the data
+//! to be maximally distributed. With higher replication, say 50 times, and a hash factor of 20,
+//! over 200,000 nodes are needed for maximal distribution.
+//!
+//! Each hash partition manages a [PartitionedTime] structure that is responsible for managing the
```

Comment:
This still says "hash partition", so I suspect you haven't pushed your changes yet.

Reply:
Hmm, I'm not seeing this any more and the comment is saying "Outdated". Possibly I hadn't pushed, I believe I have got this updated now :)

Reply:
Mmh, something is off, this is the same after refreshing the page.

Reply:
I see it in this comment view, and if I click the file link at the top of the thread's box then it takes me to the commit hash that the comment was made at and I see it.

I can't find it on the latest version of the code on the review?

Reply:
The latest version is here https://github.com/holochain/kitsune2/pull/47/files#diff-f10db395c584cade20b3f0b8cc59c314af201b415596138055cf7e77a099a6d7R8

Reply:
It's gone now.